js-exercises/is-triangle/isTriangle.js

```diff
@@ -0,0 +1,20 @@
+
+
+function isTriangle(...sides) {
+  if (sides.length > 3) {
+    throw new Error(
+      `Number of paramter passed is more than 3, Triangle has only three sides`
+    );
+  }
+
+  const isAllsidesNumber = (slide) => typeof slide == "number";
```

Comment:
Use proper camelCasing.
`isAllsidesNumber` -> `isAllSidesNumber`

Reply:
Also `slide` -> `side`

Reply:
Also `isAllSidesNumber` -> `areAllSidesNumber`. Since it is plural.

Reply:
Yes, i accept all the changes , no spelling mistakes, proper camelCasing..will change and commit back